### PR TITLE
Fix/clean code

### PR DESCRIPTION
**Pull request by raphrel**

### CODE REVIEW

File: Sources/Core/ModernAVPlayer.swift [hunk 1]
```diff
@@ -79,9 +112,16 @@ public extension ModernAVPlayer {
         case paused
         case playing
         case stopped
-        case waitingNetwork
+        case waitingForNetwork
         
-        public var description: String { return rawValue.capitalized }
+        public var description: String {
```
Comment (JiceDessaint) [inline Sources/Core/ModernAVPlayer.swift]: As you provide a description var for the enum, you can just add `CustomStringConvertible` conformance to the enum.
Even if it's not directly used, it's clearer (and free)
``` swift
enum State: String, CustomStringConvertible {
...
}
```

File: Tests/ContextSpecs.swift [hunk 2]
```diff
@@ -1,10 +1,28 @@
+// The MIT License (MIT)
 //
-//  ContextSpecas.swift
-//  RxAudioPlayerSMTests
+// ModernAVPlayer
+// Copyright (c) 2018 Raphael Ankierman <raphael.ankierman@radiofrance.com>
 //
-//  Created by raphael ankierman on 28/02/2018.
-//  Copyright © 2018 raphael ankierman. All rights reserved.
+// ContextSpecas.swift
```
Comment (JiceDessaint) [inline Tests/ContextSpecs.swift]: ContextSpecs.swift
👊

File: Sources/Core/ModernAVPlayer.swift (hunk 1 above)
Comment (raphrel) [reply Sources/Core/ModernAVPlayer.swift]: i like it

File: Tests/ContextSpecs.swift (hunk 2 above)
Comment (raphrel) [reply Tests/ContextSpecs.swift]: i like it

**Pull request merged**